packages/cli/src/index.ts

```diff
@@ -14,6 +25,27 @@ program
     initCommand();
   });
 
+program
+  .command('sync')
+  .option(
+    '-n, --novu-cloud-api-url <novuCloudApiUrl>',
+    'The Novu Cloud API url, defaults to https://api.novu.co',
+    'https://api.novu.co'
+  )
+  .requiredOption('--novu-endpoint-url <novuEndpointUrl>', 'The novu endpoint URL')
+  .requiredOption('--api-key <apiKey>', 'The Novu api key')
```

Comment:
PTAL [this](https://github.com/novuhq/novu/pull/5857)

```suggestion
  .requiredOption('--secret-key<secretKey>', 'The Novu secret key')
```